10-Sprint-Mission/css/style.css

```
@@ -0,0 +1,168 @@
+body {
+  
+}
+
+.header {
+  width: 1920px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+}
+
+.header a img {
+  max-height: 100%;
+  margin-left: 360px;
+  margin-right: 360px;
```

P3:
아래도 동일하게 작동합니다.

```suggestion
  margin: 0 360px; // 만약 중앙정렬을 하고 싶으신거라면 margin: 0 auto도 가능합니다.
```